main.js

```diff
@@ -55,4 +55,4 @@ function updateTwitterIntent(quote) {
   var intentAddress = 'https://twitter.com/intent/tweet?text=';
```

Comment:
Yeah, this section was not very well thought out.

a) `Math.ceil(0)` just returns 0, so it doesn't make any sense to use it.
b) You use `Math.floor()` when you want to round a floating point number down to the closest integer. Using it on `array.length` doesn't make sense because `array.length` will in any case be equal to an integer. 

It does make sense, however, to use `Math.floor()` on the value returned by `Math.random() * array.length` because that value will be a floating point number, and we ultimately want to round down so that we get an integer that excludes `array.length` from the result.

In the end, we don't need all the `(max - min) + min` stuff either because `min` is always 0. So we end up with: `Math.floor(Math.random() * array.length)`